content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -4185,6 +4185,547 @@ Timestamp,Name,Login,Event,Description,SourceIP,RequireOrgAdmin,RequireStackAdmi
 2021-04-11T21:09:52Z,First Last,user1,Secret Decrypted,"Decrypted secret value for stack ""demo-aws-ts-webserver/dev-user1"" (cipher text suffix: ""tbpiX4c="")",192.168.10.11,false,false,false
 ```
 
+## Deployment runners
+
+<!-- ###################################################################### -->
+
+### Register a new pool
+
+```
+POST /api/orgs/{organization}/agent-pools
+```
+
+#### Parameters
+
+| Parameter           | Type      | In    | Description         |
+|---------------------|-----------|-------|---------------------|
+| `organization`      | string    | path  | organization name   |
+| `name`              | string    | body  | pool name           |
+| `description`       | string    | body  | pool description    |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{ "name":"Production", "description": "Pool for the production account" }' \
+  https://api.pulumi.com/api/orgs/{organization}/agent-pools
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "id": "12345678-8102-447f-b246-e9ec85786e23",
+  "tokenValue": "pul-75a564ac7f3a48079a0c448c1e1ec95c4cfed141"
```

Comment:
real token? should we redact it?

Reply:
no, I used the same token it was already in that page
it is used in the Personal Access Token Organization Access Token and Team Access Token  examples